src/hostapi/asio/pa_asio.cpp

```diff
@@ -928,6 +928,29 @@ PaError PaAsio_GetAvailableBufferSizes( PaDeviceIndex device,
             PaAsioDeviceInfo *asioDeviceInfo =
                     (PaAsioDeviceInfo*)hostApi->deviceInfos[hostApiDevice];
 
+            PaAsioHostApiRepresentation *asioApi = (PaAsioHostApiRepresentation*) hostApi;
+            if ( hostApiDevice == asioApi->openAsioDeviceIndex &&
+                 asioApi->openAsioDriverInfo.bufferSizeOutdated )
+            {
+                ASIOError asioError;
+                if( (asioError = ASIOGetBufferSize(&asioApi->openAsioDriverInfo.bufferMinSize,
```

Comment:
Maybe declare some variable like:

    auto info = asioApi->openAsioDriverInfo;

then use that to shorten the long lines below.